core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -564,16 +566,18 @@ oppia.controller('StateResponses', [
           'explorationContextService', 'editorContextService',
           'explorationStatesService', 'trainingDataService',
           'AnswerClassificationService', 'focusService',
-          'RULE_TYPE_CLASSIFIER',
+          'RULE_TYPE_CLASSIFIER', 'oppiaPlayerService',
```

Comment:
I don't really understand the question, sorry. Do you mean the comment in which I said "it seems weird for general editor services to depend on oppiaPlayerService"?

If so, that's mainly a judgement call based on my understanding of the structure of the application. It should be possible to get the version information from elsewhere, and it's not something that's particularly player-related, so it doesn't seem right to get it from oppiaPlayerService. (Does that answer your question, or were you asking something different?)